packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml

```diff
@@ -20,6 +20,10 @@ spec:
     tolerations:
       {{ toYaml . | nindent 6 }}
     {{ end }}
```

Comment:
Node selector looks right now, but with tolerations you can't pass in the plain `.Values.tolerations`. Looks like you actually added that below in values.yaml but then don't use it. We should be using that one and then add-on the `fluentbit_tolerations` and `fluentd_tolerations` after

Reply:
That's a good point. I had wanted to separate them, but they both need to be included in the templating.

Reply:
Done